backend/app/models/ability.rb

```diff
@@ -36,6 +37,10 @@ def user_rights
     can %i[show], OpenCall, investor: {account_id: user.account_id}
   end
 
+  def owner_rights
+    can %i[invite], User, account_id: nil
```

Comment:
must be also approved user to invite others.

Reply:
or to be more precise, owner of approved account :)

Reply:
haha, yeah, forgot to check this, nice catch :)